docs/global_constant.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Global constants
+author: Roxane Letourneau
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Description of the `Global Constants` feature
+
+The `Global Constants` is a feature introduced in the Hangzhou protocol. Its goal is to allow users to originate larger contracts or share code between contracts by using registered constants. 
+
+This feature brings the following: 
+- A new kind of operation, named `register_global_constant`, allows users to register Micheline expressions in the global table of constants. 
+- A new primitive, called `constant`, allows contracts to reference these constants by their index.
+
+**Here is a general scenario example of using the global constant:**
+
+- Alice wants to originate a contract, but its code is so large that it does not fit the Tezos size limit for contracts.
+- Alice registers a chosen expression from her contract to the global table of constants by sending a `register_global_constant` operation to the node.
+- The precedent operation returns the index of the registered constant, which correspond to the hash of the expression (Blake2b hash + Base58 encode + prefix expr)
+- Alice replaces the newly registered expression in the code of her contract with the primitive `constant` and the corresponding hash.
+- Alice can now originate her contract has its size has been compressed.
+
+## How to register a global constant with Taquito?
+
+### Contract API
+
+A `registerGlobalConstant` method is available on the `ContractProvider` class. A `value` representing the Micheline expression to register in its JSON format is minimally required as a parameter. The `registerGlobalConstant` method returns an instance of `RegisterGlobalConstantOperation` containing a `globalAddress` member that corresponds to the index(hash) of the newly registered constant.
+
+*Note that an expression can only be registered once and will result in an error from the node if trying to register the same constant multiple times.*
+
+*Note that the conversion between Micheline and its JSON format can be achieved using the `@taquito/michel-coded` if needed.*
```

Comment:
I think it should be

 `@taquito/michel-coded`  ->  `@taquito/michel-codec`

Reply:
You are right, thanks